seed/static/seed/js/seed.js

```diff
@@ -941,6 +941,26 @@ SEED_app.config(['stateHelperProvider', '$urlRouterProvider', '$locationProvider
               });
             }
           }],
+          used_columns: ['$stateParams', 'inventory_service', function ($stateParams, inventory_service) {
```

Comment:
`used_columns` is the same as `columns` just above this except the `get_[inv_type]_columns_for_org` methods take a `true` arg and sorting isn't performed.